hw/bsp/same54xplainedpro/board.mk

```diff
@@ -13,11 +13,10 @@ CFLAGS += \
   -D__SAME54P20A__ \
   -DCONF_CPU_FREQUENCY=$(CONF_CPU_FREQUENCY) \
   -DCFG_TUSB_MCU=OPT_MCU_SAME5X \
+  -DSAME54XPLAINEDPRO=1 \
+  -DBOARD_NAME="\"Microchip SAM E54 Xplained Pro\""
```

Comment:
The extra quotes turn the defined value into a proper C string. The C string itself doesn't contain the quotes.

I see your point regarding the board define. I could argue that freeing up a PLL1 is targeted at writing board specific code as well. 

I am uncertain what is the best way to proceed here. From the TinyUSB project side I can see a desire to keep the examples board agnostic. From an application developer point of view I want to the stack to be as resource friendly and configurable as possible from the get go. Often, an application needs to identify for which particular chip it is being built, i.e. to make better use of available hardware resources.

Reply:
I fully understand your approach to have optimum set up for specific board. However, I prefer having a generic even less optimized bsp code. Eventually all same5x boards (d5035_01 and same54 xplained) will be grouped into a family much like samd51/21 when adding new board is simply define pins for LED and Button and/or Uart. This will reduce effort to add and maintain bsp. So that I could have more time to spend on the stack itself. E.g with nrf5x, I will set up the internal RC as clock source even if the board equipped with external OSC, it is just not my interest to optimize bsp performance. 

regarding `BOARD_NAME` I would prefer to have it without the string quote, you could add it later on in the usage if needed, remove it is more complicated.

Reply:
I am suprised to learn that you are not interested in BSP performance. It is my estimate that there are quite a few folks out there that will experience their respective board's performance by means of running the TinyUSB examples. Not all of them might have the technical background to realize that if performance is bad, it might not the USB stack's or the board's fault but simply due to the way the board was set up.

However, given your statement, I don't think this PR is useful to the TinyUSB project. If this is also your conclusion, feel free to close this PR.

Regarding `BOARD_NAME`, I don't quite understand your reservation. To my knowledge double quotes are a necessity here. One set of quotes prevent shell misinterpretation, the next ensures the defined value evaluates to a proper C string constant.

Reply:
> I am suprised to learn that you are not interested in BSP performance. It is my estimate that there are quite a few folks out there that will experience their respective board's performance by means of running the TinyUSB examples. Not all of them might have the technical background to realize that if performance is bad, it might not the USB stack's or the board's fault but simply due to the way the board was set up.

It is not up to this repo to get the performance out of BSP. Also the BSP API used by the tinyusb example doesn't do much except button, led and uart. I prefer to have most generic clock setup that will ensure an easy path of maintenance. Most of the time I just copied it from the vendor sdk.

> However, given your statement, I don't think this PR is useful to the TinyUSB project. If this is also your conclusion, feel free to close this PR.

Thank you for your PR.

ah you are right about the BOARD_NAME, I misinterpret it somehow. I used to mixed it up with single quote than the escape.  
```
-DBOARD_NAME='"Microchip SAM E54 Xplained Pro"'
```